data/hails.txt

```diff
@@ -329,13 +337,19 @@ phrase "friendly civilian"
 		"giving their money to"
 		"paying to listen to"
 		"so in love with"
+		"streaming"
+		"on about"
+		"obsessed with"
+		"blinded by"
 	word
 		""
 		" that group"
 		" that new group"
 		" that band"
 		" that new band"
 		" those hacks"
+		" those crooks"
```

Comment:
Crooks? If you didn't like someone's music, you might call them a hack, but a crook?